exp/anbox-cloud.md

```diff
@@ -31,62 +31,78 @@ See the following table for a comparison of features for the different variants:
 
-At the heart of Anbox Cloud sits the [**Anbox Management Service (AMS)**](https://discourse.ubuntu.com/t/about-ams/24321). **AMS** has the job to handle all aspects of the application and container life cycle (including application and image updates) while ensuring high density, performance and fast container startup times.
+#### Core stack components:
 
-A developer or system administrator will manage **AMS** through the **command line interface (AMC)** or through custom-built tools interacting with the [**AMS HTTP API**](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801).
+**Anbox Management Service** - AMS is the management tool for Anbox Cloud. AMS handles all aspects of the application and container life cycle, including application and image update, while ensuring high density, performance and fast container startup times.
 
-For example, a simple Android application testing service would provide a user-facing interface dealing with things like authentication and user management, and would communicate with the REST API to add applications or start and stop containers when a user asks to.
+Users can connect to AMS via CLI or API calls on port 8444. AMS is installed as a snap on each of the control nodes and interacts with Anbox containers, requesting and releasing resources as per demand. 
 
-Anbox Cloud can be heavily customised and extended via [**platform plugins**](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844#anbox-platform-sdk) and [**addons**](https://discourse.ubuntu.com/t/managing-addons/17759). Platform plugins and addons can be built to add specific streaming capabilities, perform operations within Android containers and much more. One example of a platform plugin is the [**Anbox WebRTC Platform**](https://discourse.ubuntu.com/t/anbox-platforms/18733) used in the Anbox Streaming Stack. Addons are ways to customise the base image by installing additional software and running scripts on different life cycle hooks.
+**Anbox Management Client** - You can choose to install AMC on other machines to [control AMS remotely](https://discourse.ubuntu.com/t/how-to-control-ams-remotely/17774), but it is generally installed together with AMS. A developer or system administrator will manage AMS through the command line interface (AMC) or through custom-built tools interacting with the AMS HTTP API.
+      
+**etcd** - etcd is the database that is used to store the data managed by AMS. It provides a reliable way to store data across a cluster of machines. It gracefully handles primary node selections during network partitions and will tolerate machine failure.
 
-### Streaming stack
+The core stack also contains the Anbox subcluster which is the central part of an Anbox Cloud deployment. This subcluster is also called “region”, which indicates that you would usually have one subcluster per geographic region that you want to serve. Each Anbox subcluster requires a machine that hosts the management layer of Anbox Cloud.
```

Comment:
We need to clean a few things up here. At best we talk through that after SU.

Reply:
- Removed the term region to avoid confusion with AWS region.
- Introduced Anbox clusters and then subclusters.
- Explained LXD cluster to be a part of Anbox subcluster. For the most part, we will only talk about Anbox subcluster and only when explaining what's in the subcluster (which is probably just this topic), we need to talk about the LXD cluster.